web/package.json

```diff
@@ -48,5 +48,8 @@
         "lint": "npm run checkTypes --workspaces --if-present && eslint . && stylelint **.css",
         "format": "eslint . --fix && stylelint --fix **.css",
         "version-seal": "cross-env ENABLE_VERSION_SEAL=true tsx packages/core/tools/set_version.ts"
+    },
+    "overrides": {
+        "ws": "^7.0.0"
```

Comment:
A `TODO` comment to "remove this when ..." would be appreciated! :pleading_face:
Otherwise, I'm fine with this.

Reply:
I agree it would be!


But alas, json

Reply:
Oh, frick, heck, right.
*looks up how much json5everywhere.org would cost...*